flutter/lib/common/widgets/peer_tab_page.dart

```diff
@@ -182,14 +201,17 @@ class _PeerTabPageState extends State<PeerTabPage>
   Widget _createPeersView() {
     final model = Provider.of<PeerTabModel>(context);
     Widget child;
-    if (model.indexs.isEmpty) {
-      child = Center(
-        child: Text(translate('Right click to select tabs')),
-      );
+    if (model.visibleIndexs.isEmpty) {
+      child = visibleContextMenuListener(Center(
+        child: Text(translate(isMobile
+            ? 'Long press to select tabs'
```

Comment:
Where does this display on mobile?

Reply:
![1acb24961aea54c2469ca9043b611b2](https://github.com/rustdesk/rustdesk/assets/14891774/2572d93c-e708-403a-be69-9ee7c9dc5fdf)